docs/index.rst

```diff
@@ -77,6 +77,7 @@ You can contribute to `pypdf on GitHub <https://github.com/py-pdf/pypdf>`_.
    meta/project-governance
    meta/history
    meta/CONTRIBUTORS
+   meta/scope-of-pypdf
```

Comment:
I do not see other pages where such information could be added, however I would have raised this into user guide section: inhere the "standard" users may not read it.